src/components/CheckboxGroup/readme.md

```diff
@@ -164,3 +164,117 @@ class CheckboxGroupTry extends React.Component {
 
 <CheckboxGroupTry />;
 ```
+
+##### Checkbox Group basic
+
+```js
+import {useState, useEffect} from 'react';
+import { CheckboxGroup } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    max-width: 500px;
+    margin: 0 auto;
+`;
+
+const Title = styled.h2.attrs(props => {
+   return props.theme.rainbow.palette;
+})`
+    font-size: 24px;
+    margin-bottom: 20px;
+    color: ${props => props.text.main}
+`;
+
+const ProfileContainer = styled.div`
+     @media (max-width: 768px) {
+        flex-direction: column;
+     }
+`;
+
+const ProfileInfoItem = styled.div`
+    margin-bottom: 15px;
+    :last-child{ margin-bottom: 0; }
+`;
+
+const ProfileLabel = styled.div.attrs(props => {
+  return props.theme.rainbow.palette;
+})`
+    font-size: 14px;
+    color: ${props => props.text.label}   
+`;
+
+const ProfileDescription = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    font-size: 16px;
+    color: ${props => props.text.main}
+`;
+
+const ProfileLeftCol = styled.div`
+    min-width: 250px;
+
+    @media (max-width: 768px) {
+        margin-bottom: 20px;
+    }
+`;
+
+const ProfileRightCol = styled.div``;
+
+const options = [
+    { value: 'push-notifications', label: 'Push Notifications', disabled: false },
+    { value: 'email', label: 'Email', disabled: false },
+    { value: 'sms', label: 'SMS', disabled: false },
+];
+const defaultOptions = ['email'];
+
+
+const CheckBoxGroupProfile = () => {
+    const [values, setSelected] = useState([]);
+    
+    useEffect( () => {
+         setSelected(defaultOptions);   
+    }, []);
+
+    return (
+        <Container className="rainbow-p-around_x-large rainbow-p-bottom_xx-large">
+            <Title>Profile</Title>
+
+            <ProfileContainer className="rainbow-flex">
+
+                <ProfileLeftCol className="rainbow-flex">
+
+                    <div>
+                        <img src="https://placeimg.com/110/110/people"/>
+                    </div>
+
+                    <div className="rainbow-p-left_medium">
+                        <ProfileInfoItem>
+                            <ProfileLabel>Name</ProfileLabel>
+                            <ProfileDescription>Jane Doe</ProfileDescription>
+                        </ProfileInfoItem>
+
+                        <ProfileInfoItem>
+                            <ProfileLabel>Company</ProfileLabel>
+                            <ProfileDescription>Nexxtway</ProfileDescription>
+                        </ProfileInfoItem>
+                    </div>
+
+                </ProfileLeftCol>
+
+                <ProfileRightCol>
+                    <CheckboxGroup
+                        id="checkbox-group-2"
+                        label="Checkbox Group Label"
+                        options={options}
+                        value={values}
+                        onChange={ (values) => setSelected(values) }
```

Comment:
if arguments passed to setSelected are the same that onChange receive then you can use:
```onChange={setSelected}```